examples/tutorials/03_vision/01_light_when_face.py

```diff
@@ -21,13 +21,12 @@
 '''
 
 import asyncio
-import sys
 import time
 
 import cozmo
 
 
-def light_when_face(robot):
+def light_when_face(robot: cozmo.robot.Robot):
```

Comment:
Looks like this script used to support having a KeyboardInterrupt. Does it still? If yes, we should have this line:

print("Press CTRL-C to quit")